pkgs/applications/emulators/ruffle/default.nix

```
@@ -19,13 +19,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "ruffle";
-  version = "nightly-2023-04-10";
+  version = "nightly-2024-01-21";
 
   src = fetchFromGitHub {
     owner = "ruffle-rs";
     repo = pname;
```

```suggestion
    repo = "ruffle";
```
See https://github.com/NixOS/nixpkgs/issues/277994.